test/step1.py

```diff
@@ -16,7 +16,7 @@
 
 process.out = cms.OutputModule("PoolOutputModule",
    fileName = cms.untracked.string("step1.root"),
-   outputCommands = cms.untracked.vstring("drop *", "keep *_*Table_*_*"),
+   outputCommands = cms.untracked.vstring("drop *", "keep *_*Table_*_*","keep edmTriggerResults_*_*_NANO","keep edmTriggerResults_*_*_HLT"),
```

Comment:
we may want to keep edmTriggerResults_*_*_RECO (for prompt or relvals) and edmTriggerResults_*_*_PAT (for re-miniAOD or MC)
alternatively we may just keep all (as in miniAOD), and make sure re-HLT and re-miniAOD campaings drop on input the old one (this was indeed done for the re-HLT caopaing in the past)

Reply:
I corrected it  already, but only in the central file... let me fix